static/include/components/board/board-analogs.md

```diff
@@ -71,7 +65,7 @@ The following properties are available for `analogs`:
 | ---- | ---- | --------- | ----------- |
 |`name` | string | **Required** | Your name for the analog reader. |
 |`pin`| string | **Required** | The pin number of the ADC's connection pin, wired to the board. This should be labeled as the physical index of the pin on the ADC.
-|`chip_select`| string | **Required** | The {{< glossary_tooltip term_id="pin-number" text="pin number" >}} of the board's connection pin, wired to the ADC. |
-|`spi_bus` | string | Optional | The `name` of the SPI bus connecting the ADC and board. Required if your board must communicate with the ADC with the SPI protocol. |
+|`chip_select`| string | **Required** | The chip select index of the board's connection pin, wired to the ADC. |
+|`spi_bus` | string | Optional | The index of the SPI bus connecting the ADC and board. |
```

Comment:
This one is still required, and not optional.

Reply:
ah yes thank you!!